caproto/server/common.py

```diff
@@ -464,9 +478,8 @@ def get_db_entry():
                         status=write_status,
                         data_count=db_entry.length
                     )
-
-                if client_waiting:
-                    await self.send(response_command)
+                    if client_waiting:
```

Comment:
might as well move this up to include creating the response?  Do we have to turn the crank an creating that even if the client is not waiting for it back?

Reply:
This is the first time in python that I have missed the extra scoping you can get with `{}` in c++...